EpiAware/src/EpiInference/EpiInference.jl

```diff
@@ -1,16 +1,21 @@
-module EpiInference
 
 """
 Module for defining inference methods.
 """
+module EpiInference
 
+using ..EpiAwareBase: AbstractEpiAwareMethod, AbstractEpiAwareOptMethod,
+                      AbstractEpiAwareSamplingMethod
 using Pathfinder: pathfinder, PathfinderResult
 
 using DynamicPPL, DocStringExtensions
 
+export AbstractNUTSMethod
```

Comment:
Logic for moving to multiple line vs single line?

Reply:
I've got a vague preference for moving export to groups depending on the object being exported e.g. function, Struct, abstract type etc.

This would seem to make the code easier to read, but its not a style I've seen strongly advocated anywhere and we'd have to refactor other bits of the code base to do it. Thoughts?

I'm also ok with going to single line exports.

Reply:
I like it. Lets use it here and add an issue to do elsewhere.